src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -18,24 +20,39 @@ public class Limelight extends SubsystemBase {
   // private double[] targetPose = null;
   private Pose3d botPose;
 
+  private final InterpolatingDoubleTreeMap shooterMap;
+
   public Limelight(Drivetrain drivetrain) {
     this.drivetrain = drivetrain;
     poseEstimator = new SwerveDrivePoseEstimator(
         drivetrain.getKinematics(),
         Rotation2d.fromDegrees(drivetrain.getHeading()),
         drivetrain.getModulePositions(),
         new Pose2d());
+
+    LimelightHelpers.SetFiducialIDFiltersOverride(SHOOTER_LL_NAME, VALID_IDS);
+
+    shooterMap = new InterpolatingDoubleTreeMap(); // add values after testing
+    shooterMap.put(0.0, 0.0); // key is distance (meters), value is angle (rads)
```

Comment:
add the comment about what the values are right above where you create the variable so it's easier to see